### ISSUE TITLE
hey

### ISSUE THREAD

**Issue by SolarAquarion** [OsmanthusTree/base16-gnome-terminal-once#1]
weren't you supposed to update it?

**Comment by OsmanthusTree** [comment]
Yes! I am glad to do it! The reason why i did not update for a long time is that i had to participate in China's college entrance examination. But i have been already admitted by a university. So I have enough time to manage this project now !

**Comment by SolarAquarion** [comment]
Congrats

**Comment by OsmanthusTree** [comment]
Thanks!
